libs/hdf-converters/src/msft-secure-score-mapper.ts

```diff
@@ -175,7 +175,8 @@ export class MsftSecureScoreMapper extends BaseConverter {
                     data: ControlScore & {implementationStatus: string}
                   ) => data.implementationStatus
                 },
-                start_time: {transformer: () => this.data.createdDateTime}
+                start_time: {transformer: () => this.data.createdDateTime},
+                run_time: 0
```

Comment:
Why is it necessary to have here explicitly?  Can't you just assume a runtime duration of 0 if this field is absent?

Reply:
Will discuss with sponsor about necessity.